embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/session/TimedSessionTest.kt

```diff
@@ -35,7 +35,7 @@ internal class TimedSessionTest {
         val clock = FakeClock(IntegrationTestRule.DEFAULT_SDK_START_TIME_MS)
         IntegrationTestRule.Harness(
             fakeClock = clock,
-            workerThreadModule = FakeWorkerThreadModule(clock),
+            workerThreadModule = FakeWorkerThreadModule(clock,SESSION_CLOSER),
```

Comment:
Shouldn't this make the linter fail? The PR is all green 😅

Reply:
It looks like there isn't a lint check for this. I adjusted the style anyway